<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishShowBar</class>
 <widget class="QWidget" name="DishShowBar">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="topWin" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#topWin
{    
	background-color: rgb(179,179,164,20%); /* 设置背景颜色 */
    border: 1px solid #c0c0c0; /* 设置边框 */
    border-radius: 5px; /* 设置圆角 */
    padding: 5px 10px; /* 设置内边距 */
}</string>
   </property>
   <widget class="QPushButton" name="buyNow_ptn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #d2b4a9; /* 设置背景颜色为浅棕色 */
    color: #ffffff; /* 设置文字颜色为白色 */
    border-radius: 20px; /* 设置边框为20像素的圆角 */
    border: 2px solid #b5877e; /* 设置边框颜色为深棕色 */
}

    /* 悬停样式 */
    QPushButton:hover {
        background-color: #b5877e; /* 设置悬停时的背景颜色 */
    }

    /* 按下样式 */
    QPushButton:pressed {
        background-color: #935f4f; /* 设置按下时的背景颜色 */
    }</string>
    </property>
    <property name="text">
     <string>立即购买</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBuyCar_ptn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #82e0aa; /* 设置背景颜色为浅绿色 */
    color: #ffffff; /* 设置文字颜色为白色 */
    border-radius: 20px; /* 设置边框为20像素的圆角 */
    border: 2px solid #58d68d; /* 设置边框颜色为浅绿色 */
}
    /* 悬停样式 */
    QPushButton:hover {
        background-color: #58d68d; /* 设置悬停时的背景颜色 */
    }

    /* 按下样式 */
    QPushButton:pressed {
        background-color: #2ecc71; /* 设置按下时的背景颜色 */
    }</string>
    </property>
    <property name="text">
     <string>加入购物车</string>
    </property>
   </widget>
   <widget class="DishIconLabel" name="DishShow_lab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Dish_lab">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="money_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="num_lab">
       <property name="styleSheet">
        <string notr="true">QLabel {
    background-color: transparent; /* 设置背景色为透明 */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DishIconLabel</class>
   <extends>QLabel</extends>
   <header>Tools/DishIconLabel/dishiconlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
